src/js/Header/Header.js

```diff
@@ -0,0 +1,11 @@
+import "./styles.scss";
+
+const Header = () => {

```

Comment:
You try to create a component in the React-like way without React connected:) Unfortunately it will not work with pure JS way. It's more likely that you component should look like this:
```suggestion
const Header = () => {
  const container = document.createElement('div');
  const header = document.createElement('h1');

  header.innerHTML = 'my-text';
  container.appendChild(header);

  return container;
}
```